docs/root/operations/cli.rst

```diff
@@ -235,3 +235,8 @@ following are the command line options that Envoy supports.
 
   *(optional)* This flag disables Envoy hot restart for builds that have it enabled. By default, hot
   restart is enabled.
+
+.. option:: --allow-unknown-fields
+
+  *(optional)* This flag disables validation of protobuf configurations for unknown fields. By default, the 
+  validation is enabled.
```

Comment:
Can you potentially add why someone might want to do this? (and why they might not)? cc @PiotrSikora @kyessenov and others who wanted this feature in place.

Reply:
Good point. My two cents. Validation is useful to catch erroneous configuration (for example misspelled fields). This is valuable because you might be running without (or misconfigured) features that the operator thought they had configured.

I'll let others comment on why someone might want to set the flag and allow unknown fields.